lib/realtime/helpers.ex

```diff
@@ -314,7 +314,7 @@ defmodule Realtime.Helpers do
   @spec kill_connections_to_tenant_id_in_all_nodes(String.t(), atom()) :: :ok
   def kill_connections_to_tenant_id_in_all_nodes(tenant_id, reason) do
     Task.async_stream(
-      Node.list(),
+      Node.list() ++ [node()],
```

Comment:
nitpick:
```suggestion
      [node() | Node.list()],
```
but list concatenation would also be fine for that case